src/pages/VotePage.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+import VoteCell from '../components/VoteCell';
+
+export default function VotePage() {
+  const [candidates, setCandidates] = useState([]);
+
+  async function getCandidates() {
+    // GET, Candidates
+    const response = await axios({
+      method: 'get',
+      url: 'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates',
+      responseType: 'stream',
+    });
+
+    const { status, data } = response;
+
+    switch (status) {
+      case 200:
+        // 투표순위대로 정렬
+        data.sort((a, b) => {
+          return b.voteCount - a.voteCount;
+        });
+        setCandidates(data);
+        break;
+      case 400:
+        alert('권한 없음');
+        break;
+      case 500:
+        alert('서버 오류');
+        break;
+      default:
+        break;
+    }
+  }
+
+  useEffect(() => {
+    getCandidates();
+  });
+
+  const candidatesList = candidates.map((person, index) => {
+    return <VoteCell key={index} vote={getCandidates} {...{ index }} {...{ person }} />;
+  });
+
+  return (
+    <Wrapper>
+      <Title>
+        <RedColor>Q. 13기 프론트엔드 팀장</RedColor>은 누구?
+      </Title>
+      <SubTitle>CEOS 프론트엔드 13기 개발팀장 투표 창입니다.</SubTitle>
+
+      <VoteBox>{candidatesList}</VoteBox>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.div`
+  background-color: #f2f2f2;
+`;
+
+const Title = styled.h3`
+  margin: 0;
+  padding: 0;
+
+  padding: 20px 10px;
```

Comment:
브라우저가 css코드를 읽을 때 위에서 아래로 읽으니까 
마지막으로 설정한 `padding`만 적용되지 않을까요??

```suggestion
  padding: 20px 10px;
```